modules/osm/node.js

```diff
@@ -33,7 +37,7 @@ osmEntity.node = osmNode;
 
 osmNode.prototype = Object.create(osmEntity.prototype);
 
-Object.assign(osmNode.prototype, {
+const prototype = {
```

Comment:
this is required so we can use `typeof prototype` in the JSDoc comment above. No runtime impact.